src/components/Navbar.jsx

```diff
@@ -45,14 +47,17 @@ function AppNavbar() {
                     className="d-inline-block align-top navbar-logo"
                 />{" "}
             </Navbar.Brand>
+            <DarkMode/>
             <Navbar.Toggle aria-controls="responsive-navbar-nav"/>
```

Comment:
i am guessing` <DarkMode/> ` is the button toggle thing in the navbar so please rename it to  <DarkModeButton /> or something

Reply:
changed to DarkModeToggle